pkg/flowkit/util/mixpanel.go

```diff
@@ -28,7 +28,10 @@ import (
 )
 
 const (
-	MIXPANEL_TRACK_URL = "https://api.mixpanel.com/track"
+	MIXPANEL_TRACK_URL     = "https://api.mixpanel.com/track"
+	MIXPANEL_QUERY_URL     = "https://mixpanel.com/api/2.0/engage"
+	MIXPANEL_PROFILE_URL   = "https://api.mixpanel.com/engage#profile-set"
+	MIXPANEL_PROJECT_TOKEN = "7af4e6f44df2c77935477ba103b3c529"
```

Comment:
This shouldn't be committed. We need to change this token due to security reasons, and make sure we don't include it in repo. How you solve this is you set the linker flag like so: `-ldflags "-X main.MIXPANEL_TOKEN $TOKEN"` and you then need to set that mixpanel token under repository secrets (you will need to reach out to SRE team to help you with that), after that is set in the repository, you then need to update build github actions to get that token during build: One example is here https://github.com/onflow/flow-cli/blob/master/.github/workflows/cd.yml#L18 Then you need to update makefile to include that on build command: `-ldflags "-X main.MIXPANEL_TOKEN $TOKEN"`. I will help you, but make sure first the secret is set on the repo and the token is changed.